libs/server/Resp/KeyAdminCommands.cs

```diff
@@ -15,7 +15,7 @@ internal sealed unsafe partial class RespServerSession : ServerSessionBase
         /// <summary>
         /// TryRENAME
         /// </summary>
-        private bool NetworkRENAME<TGarnetApi>(byte* ptr, ref TGarnetApi storageApi)
+        private bool NetworkRENAME<TGarnetApi>(int count, byte* ptr, ref TGarnetApi storageApi)
```

Comment:
Similar to above, I am not sure why count is needed here other than performing an arity or ACL permission check. We should do this outside given that at this point we have identified the CommandFlag.